examples/homepage.ipynb

```diff
@@ -181,12 +183,44 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "hvPlots will show widgets like the \"Time\" slider here whenever your data is indexed by dimensions that are not mapped onto the plot axes, allowing you to explore complex datasets much more easily than with the default plotting support.\n",
-    "\n",
+    "hvPlot offers the possibility to create [Matplotlib](https://matplotlib.org/) and [Plotly](https://plotly.com/) plots. Switching the underlying plotting library/backend is achieved with the `extension` function."
```

Comment:
Isn't 'switching' is done by `output`? I would say something like `Load the chosen plotting library with the extension function.` to be more accurate.